tool/benchmark.cc

```diff
@@ -0,0 +1,12 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "internal.h"
+
+int main(int argc, char **argv) {
+  args_list_t args;
+  for (int i = 1; i < argc; i++) {
```

Comment:
`int` -> `size_t`

Reply:
Any particular reason for this?

Reply:
Yeah:
* It's a nit.
* The type `size_t` is guaranteed to be big enough to be able to index into the maximal possible array size.
* `int` is unsigned. We don't want to go to negative indexes.

In addition (not applicable above), sometimes you want to do range checks using the index variable. If the variable is unsigned, you need to think more.

Reply:
But in this case, argc is an int and afaik it is impossible for it to be negative. I'd have to cast it to a size_t to make it work if I made `int i` type size_t, which I think just introduces clutter. Obv this isn't the biggest issue in the world but I still don't really get using size_t here...